submissions/adeyemiJuwon/css/contact.css

```diff
@@ -0,0 +1,341 @@
+    color: white;
+    line-height: 22px;
+    font-size: 18px;
+    margin-bottom: 15px;
+}
+.footer-link ul a{
+    text-decoration: none;
+    color: white;
+}
+.footer-link input{
+    width: 253px;
+    height: 44px;
+    line-height: 19px;
+    font-size: 16px;
+    color: rgb(255, 166, 95) ;
+}
+.footer-link button{
+    width: 108px;
+    height: 44px;
+    font-size:18px;
+    line-height: 22px;
+    margin-top: 9px;
+    color: rgb(255, 166, 95) ;;
+}
+footer p{
+text-align: center;
+color: #fff;
+}
+.social-link{
+    margin-top: 13px;
+    margin-left: auto;
+}
+.social h3, .footer-link h3{
+    color: white;
+}
+.social-link li{
+    display: inline-block;
+    margin-left: 12px;
+}
+
+
+/**********************************
+  RESPONSIVENESS
+  ***********************************/
+
+
+@media screen and ( max-width:1085px ){
+
+    
+   /**********************************
+  HEADING
+  ***********************************/
+    header{
+        flex-flow:wrap;
+        justify-content: flex-end;
+    
+    }
+    .navbar-link li{
+        margin: 15px;
+    }
+    
+
+/**********************************
+  PAGE: CONTACT
+  ***********************************/
+.contact{
+    flex-wrap: nowrap;
+    flex-flow: column;
+}
+.contact-us{
+    margin-top: 20px;
+}
+.contact-form fieldset{
+    margin: 20px 30px 17px 30px ;
+    padding: 0 30px;
+    border-radius: 10px;
+
+    
+}
+
+}
+
+
+
+
+@media screen and ( max-width:600px ) {
+
+
+/**********************************
+  HEADER
+***********************************/
+    header{
+        flex-direction: column;
+        align-items: flex-start;
+        
+    }
+    .navbar-link li {
+        display: block;
+        text-align: left;
+        line-height: 19px;
+        font-size: 16px;
+        margin-left:10px ;
+        color:black;
+    }
+    .logo{
+        margin-left: 10px;
+    }
+   
+    /**********************************
+  PAGE: CONTACT
+  ***********************************/
+  .contact{
+      flex-flow: column;
+  }
+  
+  .contact h3{
+    font-size: 20px;
+    line-height: 15px;
+    letter-spacing: 0.25px;
+    text-align: center;
+}
+.contact-list a{
+    font-size: 16px;
+}
+  .img-contact{
+    width: 30px;
+}
+
+.social-contact img{
+    width: 40px;
+}
+.social-contact{
+    margin: 10px auto;
+}
+.contact-form input{
+    width: 350px;
+    height: 25px;
+    border: none;
+    border-bottom: 1px solid #000;
+    font-size: 10px;
+    line-height: 15px;
+}
+.contact-form fieldset{
+    margin: 30px 10px 15px 10PX;
+    padding: 10px 20px 10px 40px;
+}
+.contact-form h3{
+    margin: 10px 30px 15px 20px;
+}
+.contact-form img{
+    display: block;
+    margin: 24px auto 0 auto;
+    width: 30px;
+}
+.contact-form label{
+    font-size: 20px;
+    line-height: 30px;
+}
+.contact-send{
+    border-radius: 4px;
+    margin: 19px 0px 32px 300px;
+}
+
+.contact-form .contact-send{
+    width: 50px;
+}
+textarea{
+    width: 350px;
+    height: 100px;
+}
+
+
+  
```

Comment:
line 327 to the end isn't properly formated

Reply:
what do you mean by not properly formated
Thanks